saskatoon/harvest/views.py

```diff
@@ -26,6 +27,22 @@ class EquipmentCreateView(PermissionRequiredMixin, SuccessMessageMixin, CreateVi
     success_url = reverse_lazy('equipment-list')
     success_message = _("Equipment created successfully!")
 
+    def get_organization(self):
+        """If creating new harvest from Organization page"""
+        try:
+            pid = self.request.GET['pid']
```

Comment:
"pid" stood for "Property ID", let's use `aid` ?

Reply:
for `actor_id`? sound good!